aptos-move/aptos-vm/src/natives.rs

```diff
@@ -76,7 +76,8 @@ pub fn assert_no_test_natives(err_msg: &str) {
                     && func_name.as_str() == "generate_keys_internal"
                 || module_name.as_str() == "bls12381" && func_name.as_str() == "sign_internal"
                 || module_name.as_str() == "bls12381"
-                    && func_name.as_str() == "generate_proof_of_possession_internal")
+                    && func_name.as_str() == "generate_proof_of_possession_internal"
+                || module_name.as_str() == "event" && func_name.as_str() == "emitted")
```

Comment:
Oh it's because you have the wrong name here. Mind changing `== "emitted"` to `.contains("emitted")`?

Reply:
addressed